src/main/pg/motor.c

```diff
@@ -61,6 +61,9 @@ void pgResetFn_motorConfig(motorConfig_t *motorConfig)
     motorConfig->minthrottle = 1070;
     motorConfig->dev.motorPwmRate = BRUSHLESS_MOTORS_PWM_RATE;
 #ifndef USE_DSHOT
+    // if (motorConfig->dev.motorPwmProtocol == PWM_TYPE_STANDARD) {
+    //     motorConfig->dev.useUnsyncedPwm = true;
+    // }
```

Comment:
Should we enable `useUnsyncedPwm` by default when `PWM_TYPE_STANDARD` is selected?

Reply:
Fixes https://github.com/betaflight/betaflight/issues/12940